docs/catalogue/cat_cohort-data-manager.md

```diff
@@ -35,232 +34,188 @@ metadata are entered in the live catalogue.
 
-#### *VariableMappings* sheet
+#### *Variable mappings* sheet
 
-Harmonisation procedures at the variable level are defined in the *VariableMappings* sheet.
+Harmonisation procedures at the variable level are defined in the *Variable mappings* sheet.
 
 | *Column name* | *Description* | *Remarks* |
 | --- | --- | --- |
-| fromDataDictionary.resource \* | Source cohort pid |  |
-| fromDataDictionary.version \* | Source data dictionary version | |
-| fromTable \* | Source table name | Tables must be predefined in the *SourceTables* sheet |
-| fromVariable | Source variable name(s) | Variables must be predefined in the _SourceVariables_ sheet; When multiple variables are mapped together use a comma-separated list, e.g. v1,v2,v3 |
-| fromVariablesOtherTables.table | Other source tables | When using variables from multiple other tables, use a comma-separated list, e.g. table1,table2,table3<sup>1</sup> |
-| fromVariablesOtherTables.name | Source variable(s) from other tables than filled out under fromTable | When using variables from multiple other tables, use a comma-separated list, the order corresponding to the order of the tables they are derived from specified under fromVariablesOtherTables.table<sup>1</sup> |
-| toDataDictionary.resource \* | Name of the target common data model  | e.g. LifeCycle_CDM, LongITools_CDM |
-| toDataDictionary.version \* | Source data dictionary version | Look up in [variable explorer](https://data-catalogue.molgeniscloud.org/catalogue/catalogue/#/variable-explorer/) |
-| toTable \* | Target table name. | Map to a table that is defined in a common data model |
-| toVariable \* | Target variable name | Map to a variable that is defined in a common data model |
-| match | Whether the harmonisation is partial, complete or NA (non-existent) | Find list to choose from in CatalogueOntologies (StatusDetails) |
+| source \* | Databank or cohort id |  |
+| source dataset \* | Source table name | Datasets must be predefined in the *Datasets* sheet in the SourceDictionary template |
+| source variables | Source variable name(s) | Variables must be predefined in the _Variables_ sheet in the SourceDictionary template; When multiple variables are mapped together use a comma-separated list, e.g. v1,v2,v3 |
+| source variables other datasets.dataset | Other source tables | When using variables from multiple other datasets, use a comma-separated list, e.g. dataset1,dataset2,dataset3<sup>1</sup> |
+| source variables other datasets.name | Source variable(s) from other datasets than filled out under source dataset | When using variables from multiple other datasets, use a comma-separated list, the order corresponding to the order of the datasets they are derived from specified under source variables from other datasets.dataset<sup>1</sup> |
+| target \* | Name of the target common data model  | e.g. LifeCycle_CDM, LongITools_CDM |
```

Comment:
is it clear for users where to find the CDM options?

Reply:
added links